exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,4 +1,7 @@
 // Declare your function first
+function divide(num1, num2){
+return num1/num2;
```

Comment:
bit of improperly indented code here, can be a pain futher down the line so it's best to get into good habits now. The Prettier exstention should be doing the work for you see https://syllabus.codeyourfuture.io/guides/code-style-guide#indent-your-code